dev-itpro/developer/properties/devenv-dataitemtableview-property.md

```diff
@@ -22,7 +22,7 @@ Sets the key on which to sort, the sort order, and the filters for the data item
 
 ## Examples
 
-In the following example, the **DataItemView** property is used to sort a table view based on the `"Entry No."` field.
+In the following example, the **DataItemTableView** property is used to sort a table view based on the `"Entry No."` field.
 
 ```AL
 DataItemTableView = SORTING ("Entry No.");
```

Comment:
What about updating the capitalization and formatting as well, for both AL examples? (My suggestion stops at the first example because I find no way to mark all of it for a change).

```suggestion
In the following example, the **DataItemTableView** property is used to sort a table view based on the `"Entry No."` field.

```AL
DataItemTableView = sorting("Entry No.");
```